app/main/RTMedia.php

```diff
@@ -1164,7 +1164,7 @@ function enqueue_scripts_styles() {
 			$url_upload = substr( $url, 0, strrpos( $url, $rtmedia_slug ) );
 			$url        = trailingslashit( $url_upload ) . 'upload/';
 		} else {
-			$url = trailingslashit( $url ) . 'upload/';
+			$url = 'upload' . trim( $url, '/' );
```

Comment:
Why prefix `upload` in url ? 
possible output `uploadhttps://yahil.rtdemo.in/?a=sds`